index.bs

```diff
@@ -895,10 +881,13 @@ APIs that require [=transient activation=] or [=sticky activation=]:
 - {{PresentationRequest/start()|presentationRequest.start()}} [[PRESENTATION-API]]
 - {{Window/showOpenFilePicker()}}, {{Window/showSaveFilePicker()}}, and {{Window/showDirectoryPicker()}} [[FILE-SYSTEM-ACCESS]]
 - {{IdleDetector/requestPermission()|IdleDetector.requestPermission()}} [[IDLE-DETECTION]]
-- Firing of clipboard events, as well as {{Clipboard/read()|navigator.clipboard.read()}} and {{Clipboard/readText()|navigator.clipboard.readText()}} [[CLIPBOARD-APIS]]
+- Firing of clipboard events. [[CLIPBOARD-APIS]]
```

Comment:
By the way, I couldn't actually find where clipboard events need activation in https://w3c.github.io/clipboard-apis/

Reply:
I think I was referring to some combination of https://w3c.github.io/clipboard-apis/#integration-with-other-scripts-and-events , plus the fact that copy/paste events only make sense in the context of a user action which can only occur in visible/interact-with-able pages.